acceptance/tests/resource/service/puppet_service_management.rb

```diff
@@ -1,6 +1,6 @@
 test_name "The Puppet service should be manageable with Puppet"
```

Comment:
not really in the scope of this PR but you could wrap `test_name` in a block and get rid of the `audit:refactor` tag since it specifically references this

Reply:
Good point about this test not running for some time. Let me take this out of the PR and submit that separately.